scripts/SysTableTex.py

```diff
@@ -92,10 +92,9 @@ def tablefragment(m,signalRegions,skiplist,chanStr,showPercent,label="",caption=
       tableline += (
           " & $\\pm "
           + str("%.2f" % m[region]["totsyserr"])
-          + r"\ ["
-          + str("%.2f" % percentage)
-          + "\\%] $       "
       )
+      if showPercent: tableline += r"\ [" + str("%.2f" % percentage) + "\\%] "
```

Comment:
```suggestion
      if showPercent: tableline += f"\ [{percentage:0.2f}\%] "
```

Reply:
Thanks @kratsg and apologies for the delay